basxconnect/core/views/person/search_person_view.py

```diff
@@ -66,13 +59,20 @@ def _display_as_list_item(person):
             hg.SPAN(
                 mark_safe(person.personnumber),
```

Comment:
We should add ```highlight.highlight``` here too in order to see matching personnumbers. I added this in another commit (which was in that PR from yesterday), so when merging with main this should be okay. But expect a small merge conflict I guess.